src/borg/archiver.py

```diff
@@ -2370,7 +2370,8 @@ def do_subcommand_help(self, parser, args):
     def preprocess_args(self, args):
         deprecations = [
             # ('--old', '--new' or None, 'Warning: "--old" has been deprecated. Use "--new" instead.'),
-            ('--noatime', None, 'Warning: "--noatime" has been deprecated because it is the default now.')
+            ('--noatime', None, 'Warning: "--noatime" has been deprecated because it is the default now.'),
+            ('--nobsdflags', None, 'Warning: "--nobsdflags" has been deprecated because it is misleading. Use --noflags instead.')
```

Comment:
remove the "because...".

it might not be true for linux users who have read the docs.
also it is for sure not be true for bsd users.